trailblazer/cli/core.py

```diff
@@ -227,7 +227,10 @@ def delete(context, analysis_id: int, force: bool, cancel_jobs: bool):
 
 @base.command("ls")
```

Comment:
just a general thought, `ls` is a reserved cmd in unix. Do we want to have a command named similar to that?

Reply:
well it would be `trailblazer ls`, that being said, I have no idea about the naming decision. In my opinion `ls` might not be the most informative for this

Reply:
Yes, with using `trailblazer ls` I'd expect  a list of files since that is what ls is supposed to do to me. But that could be a subjective thing. :)

Reply:
I agree with you. But in any case that's another PR :)